examples/network/edgeStyles/arrows.html

```diff
@@ -52,7 +52,18 @@
     nodes: nodes,
     edges: edges
   };
-  var options = {};
+var options = {
+  edges: {
+    label:'label hi',
+    font: {
+      color: '#A3A9B0',
+      highlight: '#475461',
```

Comment:
I think the arrows example is not the best place to show the new highlighted label feature. Can you please restore `edgeStyles/arrows.html` and make your changes e.g. in `labels/labelColorAndSize.html`